manifests/artemis.pp

```diff
@@ -0,0 +1,58 @@
+# Configuration for Artemis
+#
+# @api private
+class candlepin::artemis {
+  assert_private()
+
+  file { "${candlepin::catalina_home}/conf":
```

Comment:
These two directories are created by tomcat, and I think these folder definitions are conflicting with that. notice what happens to the contents of this folder with this PR:

before
```
[root@centos7-katello-artemis vagrant]# ll /usr/share/tomcat/conf/
total 216
drwxrwxr-x. 3 root   tomcat     23 Mar 17 23:49 Catalina
-rw-r--r--. 1 root   tomcat  13443 Mar 17 23:49 catalina.policy
-rw-r--r--. 1 root   tomcat   6696 Mar 17 23:49 catalina.properties
drwxr-xr-x. 2 root   tomcat     20 Apr 15 05:22 conf.d
-rw-r--r--. 1 root   tomcat   1394 Mar 17 23:49 context.xml
lrwxrwxrwx. 1 tomcat tomcat     29 Apr 14 21:35 keystore -> /etc/candlepin/certs/keystore
-rw-r--r--. 1 root   tomcat    547 Mar 17 23:49 log4j.properties
-rw-r--r--. 1 root   tomcat   3288 Mar 17 23:49 logging.properties
-rw-r--r--. 1 root   root     6835 Apr 14 21:38 server.xml
-rw-r--r--. 1 root   tomcat   1396 Apr 14 21:38 tomcat.conf
-rw-r-----. 1 root   tomcat   2418 Mar 17 23:49 tomcat-users.xml
-rw-r--r--. 1 root   tomcat 167655 Mar 17 23:49 web.xml
```
after
```
[root@centos7-katello-artemis vagrant]# ll /usr/share/tomcat/conf/
total 12
-rw-r-----. 1 tomcat tomcat  36 Apr 15 05:28 cert-roles.properties
-rw-r-----. 1 tomcat tomcat  83 Apr 15 05:28 cert-users.properties
drwxr-xr-x. 2 tomcat tomcat  23 Apr 15 05:28 conf.d
-rw-r-----. 1 tomcat tomcat 532 Apr 15 05:28 login.config
[root@centos7-katello-artemis vagrant]# 
```

Reply:
@ehelms for the most part things are looking good, but this is preventing real testing